lib/codecs/tests/data/native_encoding/schema.cue

```diff
@@ -5,19 +5,19 @@
 #Trace: {...}
 
 #Metric: {
-	name:       string
-	namespace?: string
-	tags?: {[string]: string}
-	timestamp?: #Timestamp
-	kind:       "incremental" | "absolute"
+	name:         string
+	namespace?:   string
+	tags?: {[string]: #TagValueSet}
```

Comment:
Did we call out the implementation change for this as breaking (including release highlight)? This has an impact on users upgrading Vector instances that are using this codec to communicate or when disk buffers are used.

Reply:
[Yes we did](https://github.com/vectordotdev/vector/pull/15309/files#diff-dae4977d9e08798012bd9d02b5007b79707cc24bee05cae7411bb2f4dbb7c73f).

Reply:
Oh, hang, those changes were backed out for the 0.26 release. I'll add them back here, good catch.

Reply:
Fixed